README.md

```diff
@@ -98,7 +107,39 @@ cd ~/my_ws
 catkin_make
 ```
 
-### Acknowledgements
+A typical RxROS file should include the <rxros/rxros.h> header file
```

Comment:
OK.

Reply:
@henrik7264: you could accept the suggestion by clicking the *Commit suggestion* button that is shown below the suggestion.

It's of course also perfectly fine to update things manually, but this may be convenient.